app/controllers/recipie_foods_controller.rb

```diff
@@ -0,0 +1,80 @@
+class RecipieFoodsController < ApplicationController
+  # before_action :set_recipe_food, only: %i[show edit update destroy]
+
+  # GET /recipe_foods or /recipe_foods.json
+  def index
+    @recipie_foods = Food.all
```

Comment:
- [ ] <b>REQUIRED:</b> You did not fix N+1 problem as the project requires.  The N+1 query problem is one of the common performance antipatterns in ORMs. It happens when a query is executed on each result of the previous query, in other words, when an application gets data from the database and then loop through the result of that data. This actually means that since `Food` belongs to `User` and hasMany `recpie_foods`, when the data is retrieved, the program will then go through the results to retrieve the details of the user from which that food belongs and the recpie_foods it has. So this is not good for the performance and we have to avoid it. Kindly refer to this [Article](https://dev.to/junko911/rails-n-1-queries-and-eager-loading-10eh#:~:text=What%20is%20the%20N%2B1,the%20result%20of%20that%20data.) to fix it.

Reply:
done, thanks!